UI/tests/unit/components/molecules/RipaStopResult.spec.js

```diff
@@ -172,4 +172,44 @@ describe('Ripa Stop Result', () => {
     expect(wrapper.html()).not.toContain('Verbal Warning')
     expect(wrapper.html()).not.toContain('Written Warning')
   })
+
+  it('should display a confirmation dialogue pop-up when "Contacted US Dept. of Homeland Security" is checked for legacy stops', () => {
```

Comment:
I would modify these tests to test the result of handleHomelandSecurityCheck().  Create a v1 and v2 stop, set resultsOfStop11 or 12 as needed, verify that handleHomelandSecurityCheck either sets showConfirmDialog to true, or leaves it at false.